corehq/apps/users/models.py

```diff
@@ -1529,11 +1529,35 @@ def _get_form_ids(self):
     def _get_case_ids(self):
         return CaseAccessors(self.domain).get_case_ids_by_owners([self.user_id])
 
+    def _get_deleted_form_ids(self):
+        return FormAccessors(self.domain).get_deleted_form_ids_for_user(self.domain, self.user_id)
```

Comment:
nit: shouldn't be necessary to pass in domain twice since it's already accessible inside the `FormAccessors` object